### PR TITLE
Day seven testing

### PR DESCRIPTION
**Pull request by apandey15071991**

### CODE REVIEW

File: js-exercises/bouncer/bouncer.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function bouncer(array) {
+  const result = [];
```
Comment (shadab14meb346) [inline js-exercises/bouncer/bouncer.js]: the `result` is a very general name maybe `truthyValues` or something can a better choice as it makes more sense want it is storing.

File: js-exercises/bouncer/bouncer.test.js [hunk 2]
```diff
@@ -3,5 +3,8 @@ import { bouncer } from './bouncer';
 describe('bouncer', () => {
```
Comment (shadab14meb346) [inline js-exercises/bouncer/bouncer.test.js]: great you have included a test for a lot of edge cases
